test/Directory.Build.props

```diff
@@ -31,6 +31,15 @@
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" Version="2.4.0-prerelease-63213-02" />
+    <PackageReference Include="coverlet.msbuild" Version="2.5.0" />
+    <PackageReference Include="Codecov" Version="1.1.0" />
   </ItemGroup>
-
+  <PropertyGroup Condition="'$(Coverage)' == 'true' and !$(Configuration.EndsWith('-netfx'))">
```

Comment:
💡 We can allow the `Coverage` property to control this.

```suggestion
  <PropertyGroup Condition="'$(Coverage)' == 'true'">
```

Reply:
Correct. Also the Coverage option should be moved to https://github.com/dotnet/machinelearning/blob/master/config.json and passed in in the yaml/groovy scripts.

Reply:
> Also the Coverage option … should be moved to …

This part is covered under a separate conversation above: https://github.com/dotnet/machinelearning/pull/2116#discussion_r247177985

Reply:
right but that conversation is already flagged as "Outdated" and easy to be overlooked. I shamelessly used this convo to raise awareness ;)